src/Microsoft.TestPlatform.CommunicationUtilities/Interfaces/ITestRequestSender.cs

```diff
@@ -21,6 +21,12 @@ public interface ITestRequestSender : IDisposable
         int InitializeCommunication();
 
         /// <summary>
+        /// Used for protocol version check with TestHost
+        /// </summary>
+        /// <returns>true if the version check is successful</returns>
+        bool CheckVersionWithTestHost();
```

Comment:
Datacollector will always be bundled with the test runner. So they can communicate in any protocol without any negotiation?